Remotion/Web/Development.WebTesting/HostingStrategies/DockerHostingStrategy.cs

```diff
@@ -59,6 +60,7 @@ public class DockerHostingStrategy : IHostingStrategy
           absoluteWebApplicationPath,
           port,
           dockerImageName,
+          dockerIsolationMode,
           hostname,
           is32BitProcess,
           mounts);
```

Comment:
I'm missing the using of the dockerIsolationMode parameter in IisDockerContainerWrapper resp. IDockerClient.Run.

Reply:
whoops xD